sql/connect/common/src/main/protobuf/spark/connect/types.proto

```diff
@@ -69,6 +69,8 @@ message DataType {
 
     // UnparsedDataType
     Unparsed unparsed = 24;
+
+    Time time = 28;
```

Comment:
Just for my understanding. Where is 25, 26, 27, @peter-toth ?

Reply:
```
  Variant variant = 25;
```
...
```
  // Reserved for geometry and geography types
  reserved 26, 27;
```